nntrainer/graph/network_graph.cpp

```diff
@@ -576,8 +576,22 @@ void NetworkGraph::inPlaceOptimize(const std::string &layer_type,
         continue;
 
       /** Share tensor with next layer */
-      l->net_input[0] = l->net_hidden[0];
-      prev_layer->net_hidden[loc] = l->net_hidden[0];
+      l->net_input[0] = l->net_hidden[0]; // I2 = O2
```

Comment:
Q. what is I2 O2 O1.G O2.V??

Reply:
I will add details in comments.

Reply:
Input to current (I2)
Output to current (O2)

Reply:
Added comments.